### PR TITLE
Clean QRScanner components + upgrade walletconnect

### PR DESCRIPTION
**Pull request by mikedemarais**
- fixes bug i was seeing where scanning qr codes didnt work
- uses `react-native-camera` directly instead of using the bad `react-native-qrcode-scanner` package we were using which was just a bad wrapper around `react-native-camera` anyways
- upgrades walletconnect

### CODE REVIEW

File: src/redux/walletconnect.js [hunk 1]
```diff
@@ -109,33 +109,31 @@ export const walletConnectOnSessionRequest = (
         const { peerId, peerMeta } = payload.params[0];
         const imageUrl = get(peerMeta, 'icons[0]');
         const name = convertDappNameToDisplay(peerMeta.name);
-        InteractionManager.runAfterInteractions(() => {
```
Comment (brunobar79) [inline src/redux/walletconnect.js:112]: I think this was here for a reason

Comment (brunobar79) [reply src/redux/walletconnect.js:112]: If you remove this you won't see the animation of the WC prompt sheet sliding from the bottom while doing mobile deeplinking.

Comment (mikedemarais) [reply src/redux/walletconnect.js:112]: @brunobar79 i see the animation just fine! i think this change is actually ok. also, this particular line was like the main culprit for why i was having problems scanning QR Codes sometimes

Comment (brunobar79) [reply src/redux/walletconnect.js:112]: I tested it and that’s what I saw but if you say it works then great.

**Pull request merged**